jfr-streaming/src/test/java/io/opentelemetry/contrib/jfr/metrics/MemoryLimitMetricTest.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.contrib.jfr.metrics;
+
+import static io.opentelemetry.contrib.jfr.metrics.internal.Constants.ATTR_COMPRESSED_CLASS_SPACE;
+import static io.opentelemetry.contrib.jfr.metrics.internal.Constants.ATTR_PS_EDEN_SPACE;
+import static io.opentelemetry.contrib.jfr.metrics.internal.Constants.ATTR_PS_OLD_GEN;
+import static io.opentelemetry.contrib.jfr.metrics.internal.Constants.ATTR_PS_SURVIVOR_SPACE;
+import static io.opentelemetry.contrib.jfr.metrics.internal.Constants.BYTES;
+import static io.opentelemetry.contrib.jfr.metrics.internal.Constants.METRIC_DESCRIPTION_MEMORY_LIMIT;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.opentelemetry.sdk.metrics.data.MetricData;
+import io.opentelemetry.sdk.metrics.data.SumData;
+import org.assertj.core.api.ThrowingConsumer;
+import org.junit.jupiter.api.Test;
+
+class MemoryLimitMetricTest extends AbstractMetricsTest {
+  private void check(ThrowingConsumer<MetricData> attributeCheck) {
+    waitAndAssertMetrics(
+        metric ->
+            metric
+                .hasName("process.runtime.jvm.memory.limit")
+                .hasUnit(BYTES)
+                .hasDescription(METRIC_DESCRIPTION_MEMORY_LIMIT)
+                .satisfies(attributeCheck));
+  }
+
+  @Test
+  void shouldHaveMemoryLimitMetrics() {
+
+    System.gc();
+
+    if (garbageCollectors.contains("G1 Young Generation")) {
+      // TODO: needs JFR support
+
```

Comment:
There's a handful of errant blank lines such as this, line 34, and others. Can you do a pass to clean them up?